zcash_primitives/src/transaction/sighash_v5.rs

```diff
@@ -73,41 +112,52 @@ fn transparent_input_sigdigests<A: transparent::Authorization>(
     //   c. value of the output spent by this input (8-byte little endian)
     //   d. nSequence of the input (4-byte little endian)
```

Comment:
This comment does not match the wording or the field order of https://zips.z.cash/zip-0244#s-2g-txin-sig-digest .